myuw/dao/student.py

```diff
@@ -0,0 +1,143 @@
+"""
+This module will provide a method to retrieve all user majors, class standings
+and minors along with current and rollup variables to facilitate
+determinations about user context in the case that a user is not
+enrolled for the current quarter
+"""
+from myuw.dao.enrollment import enrollment_search
+from uw_sws.term import get_current_term
+
+
+def get_student_status(request):
```

Comment:
Could you tell me why we need this function? It seems only being referenced in test.dao.

Reply:
I'm planning on using get_student_status to refactor the profile card and provide this information on the clientside.